CHANGELOG.md

```diff
@@ -14,42 +14,61 @@
 * **New Data Source:** `vcd_nsxv_application_finder` to search applications and application groups to use with a NSX-V distributed firewall ([#988](https://github.com/vmware/terraform-provider-vcd/pull/988))
 * **New Data Source:** `vcd_nsxv_application` to fetch existing application to use with a NSX-V distributed firewall ([#988](https://github.com/vmware/terraform-provider-vcd/pull/988))
 * **New Data Source:** `vcd_nsxv_application_group` to fetch existing application_group to use with a NSX-V distributed firewall ([#988](https://github.com/vmware/terraform-provider-vcd/pull/988))
+* **New Resource:** `vcd_nsxt_network_dhcp_binding` to manage NSX-T DHCP Bindings ([#1039](https://github.com/vmware/terraform-provider-vcd/pull/1039))
+* **New Data Source:** `vcd_nsxt_network_dhcp_binding` to read NSX-T DHCP Bindings ([#1039](https://github.com/vmware/terraform-provider-vcd/pull/1039))
+* **New Resource:** `vcd_nsxt_edgegateway_rate_limiting` to manage NSX-T Edge Gateway Rate Limiting ([#1042](https://github.com/vmware/terraform-provider-vcd/pull/1042))
+* **New Data Source:** `vcd_nsxt_edgegateway_rate_limiting` to read NSX-T Edge Gateway Rate Limiting ([#1042](https://github.com/vmware/terraform-provider-vcd/pull/1042))
+* **New Data Source:** `vcd_nsxt_edgegateway_qos_profile` to read QoS profiles available for
+  `vcd_nsxt_edgegateway_rate_limiting` resource ([#1042](https://github.com/vmware/terraform-provider-vcd/pull/1042))
+* New guide to install **Container Service Extension (CSE)** v4.0 in VCD 10.4+ ([#1003](https://github.com/vmware/terraform-provider-vcd/pull/1003), [#1053](https://github.com/vmware/terraform-provider-vcd/pull/1053))
```

Comment:
Could we move this line to the very top of the feature list, as it is the highlight of the release?

Reply:
done